asset/src/__lib/chunked-file-reader/formatters.ts

```diff
@@ -92,15 +92,18 @@ function csv(incomingData, logger, opConfig, metadata, slice) {
             }
             return null;
         }
-    });
+    }
+
+    return Promise.all(data.map(call));
 }
 
 // tsv is just a specific case of csv
-function tsv(incomingData, logger, opConfig, metadata, slice) {
-    return csv(incomingData, logger, opConfig, metadata, slice);
+export function tsv(incomingData: any, logger: Logger, opConfig: any, metadata: any, slice: any) {
```

Comment:
What is the point of using typescript we use `any` everywhere?
```suggestion
export function tsv(incomingData: string, logger: Logger, opConfig: FileConfig, metadata: DataEntityMetadata, slice: FIXME) {
```